fe/fe-core/src/main/java/com/starrocks/planner/PlanFragment.java

```diff
@@ -188,6 +194,22 @@ public ExchangeNode getDestNode() {
         return destNode;
     }
 
+    public List<PlanNode> getOlapScanNodes(PlanNode root) {
+        if (root instanceof OlapScanNode) {
+            return Lists.newArrayList(root);
+        }
+
+        if (root.getChildren().isEmpty()) {
```

Comment:
The  List<PlanNode> scanNodes could pass by args, then we needn't this if branch, and could reduce list object generate.

Reply:
ok